src/resources/BaseInterfaces.ts

```diff
@@ -7,6 +7,15 @@ export type PageModel<T = any, TItemsKey extends string = 'items'> = {
 
 export type New<T, K extends keyof T | null = null> = Omit<T, 'id' | NonNullable<K>>;
 
+export type NewHostedInterface<T, K extends keyof T | null = null> = Omit<
```

Comment:
Cette syntaxe est épouvantable lol est-ce que c'est pas moins compliqué d'avoir juste une interface avec name dedans sur le side? Je sait pas ce que admin-ui en pensent 👀

Reply:
C'est plus limité, pcq techniquement tu peux passer une partie de la config que tu veux, `name` est juste le seul field obligatoire !

Reply:
I can't wait for the Standards to come into effect and solves all those question and avoid this waste of time 😆

Reply:
?

Reply:
?